Output vehicle id when splitting by vehicle (#110)
Mapotempo/optimizer-api: CHANGELOG.md

## [Unreleased]

### Added

> - Corresponding vehicle_id provided for each service when using partition with vehicle entity.

Reviewer: ```suggestion
- Corresponding vehicle_id is returned for each service if problem is partitioned with vehicle entity [#110](https://github.com/Mapotempo/optimizer-api/pull/110)
```

> - Corresponding vehicle_id is returned for each service if problem is partitioned with vehicle entity [#110](https://github.com/Mapotempo/optimizer-api/pull/110)

Reviewer 2: ```suggestion
- `vehicle_id` to services `skills` when using partition with vehicle entity [#110](https://github.com/Mapotempo/optimizer-api/pull/110)
```

> - Corresponding vehicle_id is returned for each service's skills if problem is partitioned with vehicle entity [#110](https://github.com/Mapotempo/optimizer-api/pull/110)

Author: ```suggestion
- Corresponding vehicle_id is returned within each service's skills if problem is partitioned with vehicle entity [#110](https://github.com/Mapotempo/optimizer-api/pull/110)
```
